pkg/controller/cluster/resource/resource.go

```diff
@@ -350,10 +350,34 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.Cluster, si
 		},
 	}
 
+	sysctlContainer := sysctlInitContainer(c.Spec.Sysctls)
+	if sysctlContainer != nil {
+		sts.Spec.Template.Spec.InitContainers = append(sts.Spec.Template.Spec.InitContainers, *sysctlContainer)
+	}
 	sts.Spec.Template.Spec.Containers = append(sts.Spec.Template.Spec.Containers, *agentContainer(c))
 	return sts
 }
 
+func sysctlInitContainer(sysctls []string) *corev1.Container {
+	if len(sysctls) == 0 {
+		return nil
+	}
+	opt := true
+	return &corev1.Container{
+		Name:            "sysctl-buddy",
+		Image:           "busybox:1.31.1",
```

Comment:
Could this be yaml configurable too?

Reply:
We can and we can even move it to the operator image if we want.

Reply:
Oh that would be neat.

Reply:
The question is if we want it yaml configurable if we move it to the operator. We can just let the operator do it and be done. Nothing more to think about.